tests/override_config_table/utilities.py

```diff
@@ -32,6 +32,8 @@ def reload_minigraph_with_golden_config(duthost, json_data, safe_reload=True):
     golden_config = "/etc/sonic/golden_config_db.json"
     duthost.copy(content=json.dumps(json_data, indent=4), dest=golden_config)
     config_reload(duthost, config_source="minigraph", safe_reload=safe_reload, override_config=True)
```

Comment:
@liuh-80 the cleanup is done after load minigraph, will  tacacs in golden_config_db.json be enabled by default? I guess that's the purpose of this case, load minigraph is different with config reload, right? just want to double confirm if tacacs will be enabled even we remove this file.

Reply:
For nightly test, golden_config_db.json should not exist before run test.
The golden_config_db.json are generated by different test case.

For test_fallback_to_local_authorization_with_config_reload test case, the TACACS will enabled by test case, and after test finish, the config been revert by test case.

However, because this method doesn't cleanup the generated golden_config_db.json, if some other test case/human load config, the TACACS been enabled again. which should not happen.